index.js

```diff
@@ -14,6 +16,8 @@ app.set('view engine', 'pug');
 
 app.use(morgan('dev'));
 
+app.use(express.static('public'));
```

Comment:
this way express serves all the static files under public over http. Example static files: local javaScript, css files and images.